lib/client/bst-config.ts

```diff
@@ -112,7 +112,7 @@ export class BSTConfig {
         const pipeInfo = await BSTConfig.createSpokesPipe();
 
         return {
-            "sourceID": pipeInfo.endpoint.name,
+            "sourceID": pipeInfo.endPoint.name,
```

Comment:
not a tslint recommendation, but this is how it's named in spokes. So it's named like this for consistency.